crates/net/src/lib.rs

```diff
@@ -1,36 +1,43 @@
 pub(crate) mod discv4;
 use discv4::{Endpoint, FindNodeMessage, Message, Packet, PingMessage, PongMessage};
 use ethereum_rust_core::H512;
+use k256::ecdsa::{RecoveryId, Signature, VerifyingKey};
 use k256::elliptic_curve::sec1::ToEncodedPoint;
 use k256::elliptic_curve::PublicKey;
+use k256::SecretKey;
 use k256::{ecdsa::SigningKey, elliptic_curve::rand_core::OsRng};
 use keccak_hash::H256;
+use rlpx::ecies::RLPxConnection;
 use std::{
     net::SocketAddr,
     time::{Duration, SystemTime, UNIX_EPOCH},
 };
+use tokio::io::AsyncReadExt;
 use tokio::{
+    io::AsyncWriteExt,
```

Comment:
`AsyncReadExt` and `AsyncWriteExt` could be declared in the same `use` line.
`cargo fmt` does not fit these together? 🤔

Reply:
`cargo fmt` doesn't merge them by default, so I did it manually. [We might want to modify the `imports_granularity` setting for this](https://github.com/rust-lang/rustfmt/blob/master/Configurations.md#imports_granularity).